### PR TITLE
Add HTML, CSS and Linters

### PR DESCRIPTION
**Pull request by tasnimchy96**
In this branch we:
- Added index.html and style.css
- Added linters
- Documented professionally
- Added README.md file

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+test.md
+node_modules
```
Comment (wayungi) [inline .gitignore]: - [ OPTIONAL] To ignore node modules, it is better to use `node_modules/`. this means that ignore all files and folders inside the node_modules. using it without the `/` suggest that you are ignoring a file like the `test.md`. For folders, please include the `/`

File: README.md [hunk 2]
```diff
@@ -1 +1,54 @@
...
+## :handshake: Contributing
+Contributions, issues, and feature requests are welcome!
+Feel free to check the [issues page](../../issues/).
+## :star:️ Show your support
+If you like this project feel free to share with your friends.
+## :pray: Acknowledgments
+I would like to thank Microverse for this amazing exercise.
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```
Comment (wayungi) [inline README.md]: - [ ] This link is broken, please add a [LICENCE](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to your project 👨🏾‍⚕️

File: README.md [hunk 3]
```diff
@@ -1 +1,54 @@
...
+# :book: [Hello Microverse project]
+**[Hello Microverse project]** is a microverse project where I learned about the best practices and used linters to to ensure clean code.
+## :hammer_and_wrench: Built With
+### Tech Stack
+**Client**
+React.js
+**Server**
+Express.js
+**Database**
+PostgreSQL
```
Comment (wayungi) [inline README.md]: - [ ] Very good work adding the stack here. Please make sure to add coherent information. You have not used React, Express, and any database. Please delete all of them and add HTML & CSS under client ✔️

**Review by wayungi — CHANGES_REQUESTED** [review]
Hi @tasnimchy96  
image

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- [ ] Good commit messages ✔
- [ ]  Correct GitHub flow ✔️ 
- [ ] Descriptive PR body 

## Required Changes ♻️
_Check the comments under the review._
- [x]  Your linter checks are not running. Please configure the correct using this [resource](https://github.com/microverseinc/linters-config)
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .gitignore (hunk 1 above)
Comment (tasnimchy96) [reply .gitignore]: Done

File: README.md (hunk 3 above)
Comment (tasnimchy96) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (tasnimchy96) [reply README.md]: Done

File: README.md (hunk 3 above)
Comment (tasnimchy96) [reply README.md]: A message is being shown on my pull request "This branch has conflicts that must be resolved". When I try to resolve it , it say to merge. Should I merge? Could you please kindly give me a clue?

Comment (tasnimchy96) [reply README.md]: @wayungi

Comment (divinecharlotte) [reply README.md]: hello @tasnimchy96  you don't have to merge anything you only merge a PR which is approved only as you are still requesting reviews you don't need to merge anything yet you only have to resolve the conflicts

**Review by divinecharlotte — APPROVED** [review]
Hi @tasnimchy96 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To highlight 🎯

- No linter errors.✔️
- Good communication with the reviewer ✔️
- Add all the requirements. ✔️

clap


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**